goto/the_real_goto.py

```diff
@@ -34,9 +34,13 @@ def main():
     jfile = sys.argv[1]
     magic = GotoMagic(jfile)
     command = sys.argv[2]
-    args = sys.argv[3:]
+    argv = sys.argv[3:]
 
-    output, err = run_command(magic, command, args)
+    args = list(filter(lambda word: not word.startswith('-'), argv))
```

Comment:
I see now that this will filter the "commands" ´-o` and `--open`

Reply:
yes - I suppose those should be treated as **options** and sent to the commands as the options list

Reply:
Ideally we should require the following order on all commands:
```
goto <command> [<options>...] [<args>...]
```

Reply:
Or whatever order people prefer
```
goto <option> <command> <option> <option> <arg> <arg> <arg> <option>

Reply:
`goto add existing new-url` -->  Ah hoy!  ...  `goto add existing new-url -f`
`goto rename existing1 existing2` --> Ah hoy!  ...  ↑ `goto add existing1 existing2 -f` -->  Renamed.

Reply:
Yes - by the time we want to support values in options of the type:  `--option value` -- we are far better off at putting our effort into an existing cli arg parsing framework like click, fire or even the builtin argparse.

Reply:
Maybe if we keep it simple enough, we won't need complicated option handling.